### DISCUSSION TITLE
Is Git Copilot usage included in the teachers benefits?

### DISCUSSION THREAD

**Discussion by MickyRoth** [community/community#112968] [GitHub Education] [answered]
### Hi everyone,

I just started enrolling as a teacher, but not yet verified. Is **Git Copilot usage** included in the teachers benefits of the education program?

**Comment by samus-aran** [comment, marked as answer]
Hi @MickyRoth 👋 

Yes, [Free Copilot subscription is part of the benefits for verified GitHub Educators](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-teachers/about-github-global-campus-for-teachers).

**Comment by lin20260407** [comment]


**Reply by samus-aran** [reply]
Hi @weitsunglin  , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Reply by lin20260407** [reply]
thank you my freind
